### DISCUSSION TITLE
Only get the initial scrape response

### DISCUSSION THREAD

**Discussion by viktorholk** [spider-rs/spider#200] [Q&A] [answered]
Hi, 
Its a very strong library but for my use case i would like only to use the library functionalities to scrape the initial website request.

I am working on a project that is currently using the chromiumoxide library for headless chrome. But i am dealing with a website that get blocked by Akamai protection

I tested it with Spider and had no issues. Is there a way to scrape the initial request, without setting up budget, whitelists etc. To skip unnecessary scraping.

Thanks!

**Comment by j-mendez** [comment, marked as answer]
> Hi, Its a very strong library but for my use case i would like only to use the library functionalities to scrape the initial website request.
> 
> I am working on a project that is currently using the chromiumoxide library for headless chrome. But i am dealing with a website that get blocked by Akamai protection
> 
> I tested it with Spider and had no issues. Is there a way to scrape the initial request, without setting up budget, whitelists etc. To skip unnecessary scraping.
> 
> Thanks!

Hi yes, you can set the configuration limit to 1 to only perform a single page. `website.with_limit(1)`. Glad to hear the library works for you!
